nais/prod-fss.yml

```diff
@@ -21,10 +21,10 @@ spec:
   resources:
     limits:
       cpu: "4000m"
-      memory: "8096Mi"
+      memory: "1200Mi"
```

Comment:
Los kan fremdeles bruke 2GB minne i prod, så dette er nok kanskje litt for lavt?